calibration/FLORTD/CGINS-FLORTD-01155__20200108.csv

```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1155,CC_dark_counts_cdom,48,date in filename comes from dev file

```

Comment:
Note, original device file had 1152 as the serial number, which was corrected prior to the pull request.